job.go

```diff
@@ -26,6 +26,14 @@ type Build struct {
 
 	Artifacts []Artifact `json:"artifacts"`
 	Actions   []Action   `json:"actions"`
+
+	QueueId int `json:"queueId"`
+}
+
+// The summary of a build that is returned from the Job endpoint.
+type BuildShort struct {
```

Comment:
BuildShort seems to can be replaced with Build.  Would you describe why BuildShort is required?